### PR TITLE
Fix wordings

### PR DESCRIPTION
**Pull request by aviadl**
fixes https://github.com/demisto/etc/issues/11703

### CODE REVIEW

File: Integrations/integration-VirusTotal.yml [hunk 1]
```diff
@@ -781,4 +781,4 @@ script:
     description: Private API. Retrieve comments for a given resource
 hidden: false
 fromversion: 2.5.0
-releaseNotes: "Add indicators thresholds as instance parameters "
+releaseNotes: "Add indicators thresholds as instance parameters"
```
Comment (anara123) [inline Integrations/integration-VirusTotal.yml]: there was a new release so empty all releaseNotes: "-"

**Comment by aviadl** [comment]
@anara123 please check latest commit

**Comment by anara123** [comment]
@aviadl please resolve conflicts and merge

**Comment by dorsha** [comment]
Resolved. and fixed more integrations.

**Pull request merged**